src/lib/core/CHIPConfig.h

```diff
@@ -794,9 +801,17 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
  * @def CHIP_IM_SERVER_MAX_NUM_PATH_GROUPS
  *
  * @brief Defines the maximum number of path objects, limits the number of attributes being read or subscribed at the same time.
+ *
+ * The default value comes from 3path per subsctipion * 3sub per fabric * max number of fabrics, then reserve 1 read client with 9
+ * paths for each fabric.
+ *
+ * Note: CHIP_CONFIG_MAX_FABRICS = actual max number of fabrics + 1 reserved for rotation, so CHIP_CONFIG_MAX_FABRICS - 1 is the
```

Comment:
Again, I don't think the rotation thing here is right.

Reply:
Thanks for clarification, removed.